### PR TITLE
Blog post for August - bi-monthly

### PR DESCRIPTION
**Pull request by gavD**
WAT U SAYIN

### CODE REVIEW

**Comment by jenkoian** [comment]
Really good, may be worth clarifying that by bi-monthly we mean every other month, as it seems it can be used for either every 2 months or twice every month (https://en.wiktionary.org/wiki/bimonthly)

**Comment by gavD** [comment]
haha

Flammable/inflammable

OK maite I'll make that change. Owt else?


On Fri, Aug 7, 2015 at 11:15 AM, Ian Jenkins <notifications@github.com>
wrote:

> Really good, may be worth clarifying that by bi-monthly we mean every
> other month, as it seems it can be used for either every 2 months or twice
> every month (https://en.wiktionary.org/wiki/bimonthly)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/unified-diff/unified-diff.github.com/pull/116#issuecomment-128665498>
> .
>



-- 
Gavin Davies | Automation Engineer; Radify, Inc. | radify.io

**Comment by jenkoian** [comment]
Nope reads really nice :+1:

File: blog/_posts/2015-08-07-bi-monthly-format.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+title: August update - switching to every other month
+---
+
+The Unified Diff team met up recently to discuss how it's going. Jenko made graphs and everything. I ate a pizza that made me feel unusual.
+
+Anyway, we are happy with Acorn as a venue - it puts the focus on the talks.
+
+We have a challenge, however: very few people are coming forward to do talks. Whenever you see me or Warren do a talk, it's often because someone has dropped out or we simply couldn't find a speaker and we're worried you'll get sick of the sight of us!
+
+We seem to be relying on a small group of speakers who are great but we always hoped that we could encourage more people to come forward. We could contact big companies to ask them to send speakers, but when we started Unified Diff, we wanted to be a community event - by local software people for local software people, talking about real things that we do - rather than sitting there being sold something by a tech evangelist.
```
Comment (rodnaph) [inline blog/_posts/2015-08-07-bi-monthly-format.md]: we wanted *it* to be

**Comment by rodnaph** [comment]
Lush.

**Comment by gavD** [comment]
mergin. Hope Crage not mad!!!!!!

**Pull request merged**